internal/revision.go

```diff
@@ -207,6 +207,9 @@ func MustParseInt(value string) int {
 
 func RandomString() string {
 	buf := make([]byte, 6)
-	rand.Read(buf)
+	_, err := rand.Read(buf)
+	if err != nil {
+		return ""
```

Comment:
This should panic if we cannot generate a random string. This generates important IDs for yoke, hence empty string is not a valid return.